sunpy/net/download.py

```diff
@@ -105,9 +105,15 @@ def _default_error_callback(self, e):
         raise e
 
     def wait(self):
+        """
+        Waits for all files to download completely and then return.
```

Comment:
I'm not sure that `completely` needs to be stated.

Reply:
Fixed it.